app/serializers/technology_serializer.rb

```diff
@@ -1,3 +1,14 @@
 class TechnologySerializer < ActiveModel::Serializer
   attributes :id, :name
+
+  def name
+    tech_capitalizations[object.name] ?
+    tech_capitalizations[object.name] : object.name.capitalize
+  end
+
+  def tech_capitalizations
+    {
+      "javascript" => "JavaScript",
```

Comment:
Looks great! I'm gonna wait for the checks to complete then I'll merge it. @hhoopes